draft-ietf-quic-recovery.md

```diff
@@ -558,6 +545,23 @@ keys are discarded, the PTO and loss detection timers MUST be reset, because
 discarding keys indicates forward progress and the loss detection timer might
 have been set for a now discarded packet number space.
 
+#### Before Address Validation
+
+Until the server has validated the client's address on the path, the amount of
+data it can send is limited to three times the amount of data received,
+as specified in Section 8.1 of {{QUIC-TRANSPORT}}. If no data can be sent,
```

Comment:
```suggestion
as specified in Section 8.1 of {{QUIC-TRANSPORT}}. If no additional data can be sent,
```

When you arm the timer, you might be sending a packet that is within the limit, but the next packet might exceed the limit.

I don't know if this should be adjusted somehow, but this might help.